cmake/template/setup.sh.in

```diff
@@ -22,3 +24,8 @@ export GAZEBO_RESOURCE_PATH=${GAZEBO_RESOURCE_PATH}:${ROBOTOLOGY_SUPERBUILD_INST
 # ROBOTOLOGY_ENABLE_DYNAMICS-specific lines
 export YARP_DATA_DIRS=$YARP_DATA_DIRS:${ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX}/share/codyco
 @endcmakeif ROBOTOLOGY_ENABLE_DYNAMICS
+
+@cmakeif ROBOTOLOGY_USES_PYTHON
+# Add the python bindings directory to the PYTHON_PATH
+PYTHONPATH=${PYTHONPATH}:${ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX}/lib/python2.7/dist-packages
```

Comment:
@diegoferigo Sorry, why doesn't this line contains the `export` command? For me When I ran `import yarp` in a Python interpreter it gave me an error. Then I added:
```
export PYTHONPATH=${PYTHONPATH}:${ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX}/lib/python2.7/dist-packages
```
in `.bashrc` and `import yarp` is accepted by Python now.

Reply:
The `export` should be there, you're right. No clue honestly why it's missing.

Reply:
I think it was just an error on my side. Thanks for  noticing @HosameldinMohamed .